### ISSUE TITLE
Chili only displayed on secondary monitor

### ISSUE THREAD

**Issue by freeshrugsxd** [MarianArlt/kde-plasma-chili#22]
Hey,

the login screen is only displayed on what I consider my secondary monitor. I've had similar problems with other software so I believe the reason is the hardcoded output priority of my GPU. What I consider secondary is considered the primary display because it's plugged into DVI, which has a higher priority than HDMI, which is where my primary monitor is plugged into. Long story short: Is there a way to configure chili to be displayed on a certain display? Or maybe just to mirror it across all displays?

Thanks in advance and keep up the good work! Love this project.

**Comment by MarianArlt** [comment]
@freeshrugsxd 

1. I take it you only see output on the monitor connect to the DVI port?
2. The monitor connected to HDMI stays black?
3. Is this true only for my chili theme or for other themes as well?
4. Maybe [try to enable HiDPI](https://wiki.archlinux.org/index.php/SDDM)?

Number three is the most important one to me.

Also I suppose your so called "primary" monitor with HDMI is connected to a dedicated GPU and your DVI is coming from the motherboard? If this is the case you may want to go to your BIOS and change your primary graphics to GPU instead of IGP (integrated) which _could_ set the right prioritization in your case. Worth a try.

**Comment by freeshrugsxd** [comment]
Hey, thanks for your reply!

During boot, everything is mirrored on both monitors, except for the login screen. The login screen is displayed only on the DVI monitor. The HDMI monitor goes into standby until I entered my password and becomes active again.

I just tested a few other themes, including the default one, and they all only show on the DVI monitor. This comes a bit as a surprise to me, actually! I've been using chili for quite some time and I simply assumed that the vanilla login screen would show on both monitors (because the Lock Screen does) but that does not seem to be the case. 

I guess that makes it not your problem anymore and I will have to dig around in the kde/sddm configs.

Just for further clarification though: My monitors are both connected to the same GPU. The display priority is [hardcoded into the GPU](https://www.evga.com/support/faq/afmviewfaq.aspx?faqid=59560) and there is almost nothing I can do to change the order except for tinkering with the GPU BIOS, which seems a bit overkill. Especially so since I can see everything else on my primary monitor because of how I configured my mobo's BIOS. My second monitor is not always turned on, you see? But I'm forced to turn it on to see the login screen _or_ I enter my password blind, which also works, but is just a tad inconvenient. And this is really all this is, a minor inconvenience, but I thought I would still ask you for help!

**Comment by freeshrugsxd** [comment]
Got it working!

What I did was
`$ sudo vi /etc/sddm.conf`

added the following lines:
```
[X11]
DisplayCommand=/usr/share/sddm/scripts/Xsetup
```

`$ sudo vi /usr/share/sddm/scripts/Xsetup`

and added:
`xrandr --output HDMI-0 --auto --primary --left-of DVI-I-0`

Now I got the login screen on both monitors, like I wanted! If I get tired of it I can just add
`xrandr --output DVI-I-0 --off`

to only have it displayed on my primary monitor!
